CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/gui/VisualGraphOpener.java

```diff
@@ -121,7 +121,17 @@ public void openGraph(final GraphDataObject gdo) {
                         // Backup the current actual file and replace it with the autosave file.
                         final File autosaved = new File(AutosaveUtilities.getAutosaveDir(), props.getProperty(AutosaveUtilities.ID) + FileExtensionConstants.STAR);
                         try {
-                            AutosaveUtilities.copyFile(autosaved, f);
+                            // make temp copy of any backup file that exists to try if we find both autosave and main file are corrupt
+                            final File toBak = new File(f.getPath() + FileExtensionConstants.BACKUP);
+                            if (toBak.exists()) {
+                                // Make a backup of our backup which is about to be overwrtten by the Autodsave call
+                                final File toBakBak = new File(toBak.getPath() + FileExtensionConstants.BACKUP);
+                                final boolean toRenamed = toBak.renameTo(toBakBak);
```

Comment:
The variable `toRenamed` is unused.
Should this be checking if the rename operation was successful?

Reply:
Appears to be used in the line below it